elevate-cpanel

```diff
@@ -3536,8 +3525,15 @@ sub install_cpanel_elevate_service {
 
 # TODO: We're going to need to store reboot time so we know if the last reboot has happened when we re-run the script.
 
-sub read_stage_file {
-    return eval { Cpanel::JSON::LoadFile(ELEVATE_STAGE_FILE) } // {};
+sub read_stage_file ( $k = undef ) {
+    my $stage_info = _read_stage_file() // {};
+
+    return $stage_info->{$k} if defined $k;
```

Comment:
it's not a big deal but I wonder if you should be stringifying it? 
`$stage_info->{"$k"}`